single.html

```diff
@@ -3,65 +3,140 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Example tea</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Acme&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+  <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/single_style.css">
+  <title>Greenfield black tea</title>
 </head>
 
 <body>
 
   <header>
     <div class="container">
-      <a href="#" class="header__logo">
-        <img src="img/logo220.png" alt="">
+      <a href="index.html" class="header__logo">
+        <img src="img/logo_desctop.png" alt="">
       </a>
       <div class="header__bottom">
+        <div class="header__burger"><span></span></div>
         <nav>          
-          <a href="#" class="header__button">Green tea</a>
-          <a href="#" class="header__button">Black tea</a>
-          <a href="#" class="header__button">Other</a>
-          <input type="search" placeholder="Search">
+          <a href="index.html" class="header__button" data-text="Home">Home</a>
+          <a href="gallery.html" class="header__button" data-text="Gallery">Gallery</a>
+          <a href="add_tea.html" class="header__button" data-text="Home">Add tea</a>
         </nav>
-        <div class="header__user__menu">
-          <a href="#" class="header__button">Log In</a>
-          <a href="#" class="login__logo">
-              <img src="img/login.png" alt="login">
-          </a>
+        <div class="header__login">
+          <a href="#" class="header__button" id="login">Log In</a>
         </div>
       </div>
     </div>
   </header>
 
+  <div class="popup">
+    <div id="popup-content">
+      <img src="img/login.svg" alt="User">
+      <input type="text" placeholder="Email">
+      <input type="text" placeholder="Username">
+      <input type="text" placeholder="Password">
+      <label for="">Registration complete</label>
+      <div class="popup__buttons">
+        <button id="login-button"><span>log in</span></button>
+        <button id="registration-button"><span>sign up</span></button>
+        <button id="reg-complete"><span>complete</span></button>
+     </div>
+      <div id="close-login">+</div>
+    </div>
+
+    <div id="comment-popup">
+      <div id="new-comment-stars">
+        1-5 stars here
+      </div>
+      <textarea placeholder="Your comment here..." cols="30" rows="5"></textarea>
+      <button id="comment-button"><span>leave comment</span></button>
+      <div id="close-comment">+</div>
+    </div>
+  </div>
+
   <main>
     <div class="container">
+
       <section class="tea__section">
         <div class="main__information">
```

Comment:
`article`?

Reply:
```suggestion
        <article class="main__information">
```
Will update it soon.

Reply:
Still didnt get. Should it be like section>article? I thought that if we are talking about a description of one instance, the article does not make much sense.

Reply:
Article encloses self-sufficient piece of document, which is, in your case, a variety of tea.
While section is just an arbitrary section of document with more or less distinguishable topic.

Let's say you need to create main page of your site. You want to have 'tea of the day' on top of your page and a list of recently added teas.

The structure may look like this:

```HTML
<header>
<main>
  <section id="tea-of-the-day">
    <h2>Tea of the day</h2>
    <article class="tea">
    </article>
  </section>
  <section id="last-added">
    <h2>Recently added teas</h2>
    <ol>
      <li>
        <article class="tea">
        </article>
      </li>
    </ol>
  </section>
</main>
```

Reply:
Ok. Then, what's the point of making the article on the page where I have a description of one tea i describe one tea?